Assets/tiny_rogue/Scripts/CreatureLibrary.cs

```diff
@@ -223,7 +227,7 @@ public void FixupSpritePlayer(EntityCommandBuffer cb, Entity player)
             
             cb.AddComponent(player, att);
             cb.AddComponent(player, c);
-            cb.AddComponent(player, sight);
+            //cb.AddComponent(player, sight);
```

Comment:
The whole FixupSpritePlayer can be removed

Reply:
Will do!